examples/rpl_udp/helper.c

```diff
@@ -148,7 +150,9 @@ void rpl_udp_ignore(int argc, char **argv)
     tcmd.transceivers = TRANSCEIVER_CC1100;
     tcmd.data = &a;
 
-    printf("sending to transceiver (%" PRIkernel_pid "): %u\n", transceiver_pid,
-           (*(uint8_t *)tcmd.data));
+    a = atoi(argv[1]);
+    printf("sending to transceiver (%u): %u\n", transceiver_pid, (*(uint8_t *)tcmd.data));
```

Comment:
Don't get the point of this change, nor how it is related to return based error-handling.